lib/dor/workflow/client/lifecycle_routes.rb

```diff
@@ -57,15 +63,13 @@ def filter_milestone(lifecycle_doc, milestone_name)
         #     <milestone date="2010-06-15T16:08:58-0700">released</milestone>
         #   </lifecycle>
         #
-        def query_lifecycle(repo, druid, active_only: false, version: nil)
+        def query_lifecycle(repo, druid, active_only:, version: nil)
```

Comment:
This will require a major version bump, right? (No problem.)

Reply:
Nope, this is a private method.

Reply:
Did not see that. Noice.